conf/carbon.conf.example

```diff
@@ -262,6 +262,14 @@ RELAY_METHOD = rules
 # datapoint to more than one machine.
 REPLICATION_FACTOR = 1
 
+# If your REPLICATION_FACTOR >=2, you are using more than one replica per server
+# and you want to be sure that every replica of data is located on different servers
+# please setup DIVERSE_REPLICAS to True.
+# Before carbon v0.9.14 it was not guaranteed!
```

Comment:
e.g. "the equivalent behavior for Carbon 0.9.12 was `DIVERSE_REPLICAS = False` but before 0.9.11 it was `DIVERSE_REPLICAS = True`". Note that I have **no idea** what the actual behavior was, just giving examples.